### PR TITLE
feat: Add last_login_at and password_reset_at to account API

### LINKED ISSUE #194: Add `last_login_at` and `password_changed_at` to GetAccounts
Hi @cainlevy, 

I'm writing an invitation system that will use the password reset functionality. Basically we store a status on our `User` record that tells us if this person needs an invite or if they should get a password reset and then just send them slightly different email copy and a different landing page where they set/change their password.

Part of the requirements for this feature from our client is to send invitation reminders if they have not logged in yet. To easily gather this data, I'd really like to add `last_login_at` and `password_changed_at` to the `/accounts/:id` endpoint so that we can check these values. 

I'd like to add those values to the GetAccount endpoint so that our application can check and see if the user has logged in and when they reset their password.

### PR DESCRIPTION
**Pull request by xcskier56**
### Whats Up
Add `last_login_at` and `password_reset_at` to Account API. This closes #194 

FYI, I'm very new to go, so please pardon any formatting or syntax oddities. I'm used to ruby and so I assume there's a simpler way to implement what I did, but just don't know Go very well

### CODE REVIEW

File: server/handlers/get_account.go [hunk 1]
```diff
@@ -27,11 +27,27 @@ func GetAccount(app *app.App) http.HandlerFunc {
...
+			formattedLastLogin = ""
+		} else {
+			formattedLastLogin = account.LastLoginAt.Format("2006-01-02T15:04:05Z07:00")
+		}
+
+		var formattedPasswordChangedAt string
+		if account.PasswordChangedAt.IsZero() {
+			formattedPasswordChangedAt = ""
+		} else {
+			formattedPasswordChangedAt = account.PasswordChangedAt.Format("2006-01-02T15:04:05Z07:00")
```
Comment (cainlevy) [inline server/handlers/get_account.go:41]: i went looking for a standard iso 8601 formatter and discovered:

```suggestion
			formattedPasswordChangedAt = account.PasswordChangedAt.Format(time.RFC3339)
```

**Review by cainlevy — CHANGES_REQUESTED** [review]
looks good! would you add a short mention to the CHANGELOG.md?

> I assume there's a simpler way to implement what I did

... says every Rubyist the first time they write Go 😄

File: server/handlers/get_account.go [hunk 2]
```diff
@@ -27,11 +27,27 @@ func GetAccount(app *app.App) http.HandlerFunc {
 			panic(err)
 		}
 
+		var formattedLastLogin string
+		if account.LastLoginAt == nil {
+			formattedLastLogin = ""
+		} else {
+			formattedLastLogin = account.LastLoginAt.Format("2006-01-02T15:04:05Z07:00")
+		}
```
[the remark this answers is not in the document]
Comment (xcskier56) [reply server/handlers/get_account.go:35]: 👍

File: server/handlers/get_account.go (hunk 1 above)
Comment (xcskier56) [reply server/handlers/get_account.go]: 👍

**Comment by xcskier56** [comment]
@cainlevy, what's the process for getting another release? Can I help? Unfortunately this update is a blocker for a feature that we're anxiously awaiting. Thank you!

**Review by cainlevy — APPROVED** [review]

**Comment by cainlevy** [comment]
released in v1.14.0

**Pull request merged**